CHANGELOG.md

```diff
@@ -1,268 +1,1947 @@
+-   fix tests ([#1315](https://github.com/webpack/webpack-cli/pull/1315))
+-   add tests for loader command ([#1476](https://github.com/webpack/webpack-cli/pull/1476))
+-   remove the redundancy extractSummary method for entry test ([#1313](https://github.com/webpack/webpack-cli/pull/1313))
+-   add test for zero config ([#1418](https://github.com/webpack/webpack-cli/pull/1418))
+-   add tests for init ([#1419](https://github.com/webpack/webpack-cli/pull/1419))
+-   add tests for test utils ([#1451](https://github.com/webpack/webpack-cli/pull/1451))
+-   add more tests cases for serve ([#1446](https://github.com/webpack/webpack-cli/pull/1446))
+-   add test for entry flag ([#1445](https://github.com/webpack/webpack-cli/pull/1445))
+-   webpack-cli: remove intrusive snapshot ([#1293](https://github.com/webpack/webpack-cli/pull/1293))
+-   for snapshot config related flags ([#1789](https://github.com/webpack/webpack-cli/pull/1789))
+-   generators: added style support tests ([#1471](https://github.com/webpack/webpack-cli/pull/1471))
+-   generators: languageSupport ([#1447](https://github.com/webpack/webpack-cli/pull/1447))
+-   generators: Add test case for production CSS config ([#1473](https://github.com/webpack/webpack-cli/pull/1473))
+-   generators: plugins util tests ([#1448](https://github.com/webpack/webpack-cli/pull/1448))
+-   loader-generator: add tests for loader generator ([#1234](https://github.com/webpack/webpack-cli/pull/1234))
+-   plugin & loader: refactor tests ([#1787](https://github.com/webpack/webpack-cli/pull/1787))
+-   sourcemap: fix flaky stats statement ([#1232](https://github.com/webpack/webpack-cli/pull/1232))
+-   webpack-cli: correct descriptions for no-mode test cases ([#1319](https://github.com/webpack/webpack-cli/pull/1319))
+-   webpack-cli: add case for mode option none in ZeroConfigGroup ([#1301](https://github.com/webpack/webpack-cli/pull/1301))
+-   webpack-cli: add test case for defaults flag ([#1254](https://github.com/webpack/webpack-cli/pull/1254))
+-   webpack-cli: add tests for --target flag ([#1548](https://github.com/webpack/webpack-cli/pull/1548))
+-   webpack-cli: executer tests ([#1280](https://github.com/webpack/webpack-cli/pull/1280))
+-   webpack-cli: correct node-test ([#1455](https://github.com/webpack/webpack-cli/pull/1455))
+-   webpack-cli: fix folder structure, add more default tests ([#1258](https://github.com/webpack/webpack-cli/pull/1258))
+
+## Misc
+
+-   feat[utils]: opt to use config schema from core ([#1655](https://github.com/webpack/webpack-cli/pull/1655))
+-   migrate to commander ([#1481](https://github.com/webpack/webpack-cli/pull/1481))
+-   Fix loader-generator and plugin-generator tests ([#1250](https://github.com/webpack/webpack-cli/pull/1250))
+-   Fixing the typos and grammatical errors in Readme files ([#1246](https://github.com/webpack/webpack-cli/pull/1246))
+-   remove code: remove unused code ([#1800](https://github.com/webpack/webpack-cli/pull/1800))
```

Comment:
Let's remove `chore` and `tests` from here, it is unnecessary trash and no one needs it

Reply:
no

Reply:
webpack has `chore` or `tests` in changelog or other packages use it? How you can read this bug changelog and see what is real changes?